app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_ = command.split()
```

Comment:
The variable name `list_` is not descriptive. It would be better to use a more meaningful name to represent the purpose of this variable. For example, `command_parts` would be more descriptive.